qiskit_dynamics/solve.py

```diff
@@ -49,9 +49,21 @@
 general sense (not in *standard form*) can be restructured into one of standard form via suitable
 vectorization.
 
-The function :meth:`~qiskit_dynamics.de.solve_lmde` solves LMDEs in standard form, specified
-in terms of a representation of the generator :math:`G(t)`, either as a ``Callable`` function
-or a subclass of :class:`~qiskit_dynamics.models.generator_models.BaseGeneratorModel`.
+The function :meth:`~qiskit_dynamics.de.solve_lmde` provides access to solvers for LMDEs in
+standard form, specified in terms of a representation of the generator :math:`G(t)`,
+either as a Python ``Callable`` function or subclasses of
+:class:`~qiskit_dynamics.models.generator_models.BaseGeneratorModel`.
+
+Note that methods available via :meth:`~qiskit_dynamics.solve_ode` are also available through
```

Comment:
Makes sense - will at the very least change this to be more specifically referring to _ode methods_ or _numerical methods_.

Reply:
Changed to "numerical methods". I don't know that I want to get rid of the word "method" since it's literally the name of the kwarg and is how they are referred to in the ode solving literature.